### DISCUSSION TITLE
Non posso cambiare la larghezza delle piste

### DISCUSSION THREAD

**Discussion by Betureee** [freerouting/freerouting#197] [Q&A]
Sto lavorando ad un pcb e ho installato l'ultima versione del tool dell'autorouting su KiCAD 6.0, il problema è che quando clicco per aprire la finestra di autorouting fa 1 o 2 secondi di caricamento e parte subito a fare l'autorouting, così non posso cambiare le misure di piste e vie, il problema è che se lo cambio una volta finito il routing le piste fanno corto tra loro, in sostanza non posso cambiare nessuna impostazione, quando parte il tool fa tutto con opzioni di default, secondo me un altra cosa molto scomoda è cliccare con il tasto sinistro del mouse per stoppare il tool, sarebbe più comodo che si aprisse la finestra di autorouting e ci siano tutte le opzioni per il routing, e poi poterlo avviare o stoppare dalla barra dei menù, sapreste dirmi come posso fare per cambiare la larghezza delle piste prima dell'autorouting, spero che questi consigli possano aiutare a migliorare il tool:)

**Comment by andrasfuchs** [comment]
English translation:
`I'm working on a pcb and I installed the latest version of the autorouting tool on KiCAD 6.0, the problem is that when I click to open the autorouting window it does 1 or 2 seconds of loading and starts autorouting right away, so I can't change the measurements of tracks and vias, the problem is that if I change it once the routing is finished, the tracks become short of each other, basically I can't change any setting, when the tool starts it does everything with default options, in my opinion another very inconvenient thing is to click with the left mouse button to stop the tool, it would be more convenient if the autorouting window opens and there are all the options for routing, and then being able to start or stop it from the menu bar, you know tell me how can I change the width of the tracks before autorouting, I hope these tips can help improve the tool :)`
